### PR TITLE
feat: rebalance flow

### PR DESCRIPTION
**Pull request by limpbrains**
All data is currently fake. It is just to add dashed line to PieChart component and a place for a future rebalance logic

https://user-images.githubusercontent.com/155891/192829949-920c1937-fc77-4b4e-8a15-29e127a4368f.mp4

### CODE REVIEW

File: src/screens/Lightning/RebalanceResult.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+const Result = ({
+	navigation,
+}: LightningScreenProps<'RebalanceResult'>): ReactElement => {
+	const colors = useColors();
+
+	return (
+		<GlowingBackground topLeft={colors.purple}>
+			<SafeAreaInsets type="top" />
+			<NavigationHeader title="Rebalance Succesful" displayBackButton={false} />
```
Comment (pwltr) [inline src/screens/Lightning/RebalanceResult.tsx]: small typo:

```suggestion
			<NavigationHeader title="Rebalance Successful" displayBackButton={false} />
```

File: src/screens/Lightning/RebalanceResult.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			<View style={styles.root}>
+				<View>
+					<Display color="purple">Funds{'\n'}Transferred.</Display>
+					<Text01S color="gray1" style={styles.text}>
+						You have succesfully rebalanced your savings and instant spending
```
Comment (pwltr) [inline src/screens/Lightning/RebalanceResult.tsx]:
```suggestion
						You have successfully rebalanced your savings and instant spending
```

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Wallets/WalletsDetail/BitcoinBreakdown.tsx [hunk 3]
```diff
@@ -70,14 +70,11 @@ const BitcoinBreakdown = (): ReactElement => {
 				<TouchableOpacity
 					onPress={(): void => {
 						navigation.navigate('LightningRoot', {
-							screen: 'Introduction',
+							screen: 'RebalanceSetup',
 						});
 					}}>
```
Comment (pwltr) [inline src/screens/Wallets/WalletsDetail/BitcoinBreakdown.tsx]: Do we maybe need to check here if the user has setup Lightning yet, and depending on it navigate to either `Introduction` or `RebalanceSetup`?

Comment (limpbrains) [reply src/screens/Wallets/WalletsDetail/BitcoinBreakdown.tsx]: Nice catch! I added same check as in lightning todo card

**Review by pwltr — APPROVED** [review]

**Pull request merged**